pkg/config/config.go

```diff
@@ -150,19 +150,20 @@ type Install struct {
 	VipHwAddr string `json:"vipHwAddr,omitempty"`
 	VipMode   string `json:"vipMode,omitempty"`
 
-	ForceEFI      bool   `json:"forceEfi,omitempty"`
-	Device        string `json:"device,omitempty"`
-	ConfigURL     string `json:"configUrl,omitempty"`
-	Silent        bool   `json:"silent,omitempty"`
-	ISOURL        string `json:"isoUrl,omitempty"`
-	PowerOff      bool   `json:"powerOff,omitempty"`
-	NoFormat      bool   `json:"noFormat,omitempty"`
-	Debug         bool   `json:"debug,omitempty"`
-	TTY           string `json:"tty,omitempty"`
-	ForceGPT      bool   `json:"forceGpt,omitempty"`
-	Role          string `json:"role,omitempty"`
-	WithNetImages bool   `json:"withNetImages,omitempty"`
-	WipeDisks     bool   `json:"wipeDisks,omitempty"`
```

Comment:
Do we need any doc to indicate that `harvester.install.wipe_disks` is now deprecated? Presumably, with this change, all existing references to `harvester.install.wipe_disks` won't work anymore, right?